### ISSUE TITLE
Get web server dependencies as part of an install process

### ISSUE THREAD

**Issue by gurbirkalsi** [distributed-system-analysis/pbench#475]
Use a package manager (npm, yarn) to create a module for installation of the core dependencies needed to run the pbench UI dashboard.

**Comment by ndokos** [comment]
We need to add the packages that you need to the RPM postinstall phase. Is the UI server a good example of what you need? If so, I can update the spec file.

**Comment by gurbirkalsi** [comment]
Yes, everything under the /v0.2/components/ folder are dependencies that are being used within the dashboard. We'll need these install commands:

npm install boostrap
npm install datatables.net
npm install font-awesome
npm install html5shiv
npm install jquery
npm install patternfly

There is another dependency called Respond.js which does not have an npm install however we can include that within /v0.2/js/

**Comment by portante** [comment]
@gurbirkalsi, where is the main source repository for `Respond.js`?

**Comment by gurbirkalsi** [comment]
@portante https://github.com/scottjehl/Respond

**Comment by gurbirkalsi** [comment]
List of current node modules with appropriate versions: 

bootstrap@3.3.7
datatables@1.10.13
jquery@3.2.1
patternfly@3.23.0
